src/Content/Version.php

```diff
@@ -33,39 +35,46 @@ public function content(string $language = 'default'): Content
 	{
 		return new Content(
 			parent: $this->model,
-			data:   $this->model->storage()->read($this->id, $language),
+			data:   $this->model->storage()->read($this->id, $this->language($language)),
```

Comment:
It's all a bit of a chicken and egg situation in between the different PRs. It would have been correct to still pass a string argument here, but since `Version::content` isn't used in our own code so far, it won't break anything. 

If we want to keep it temporarily correct, this would need to be fixed and then later reverted. I guess it's more a question if we are able to keep track of it until PR 9. I basically used the CMS unit tests as a safety net here.

Reply:
`Correction`: The Language object is stringable. That's why it all still keeps working with the outdated ContentStorage class.